packages/alfa-rules/src/common/predicate/has-border.ts

```diff
@@ -17,9 +17,6 @@ export function hasBorder(
     return sides.some(
       (side) =>
         style.computed(`border-${side}-width` as const).none(Length.isZero) &&
-        style
-          .computed(`border-${side}-style` as const)
-          .none((style) => style.value === "none") &&
```

Comment:
Why was this removed? `border-left-style: none` will result in there being no left border regardless of the other `border-left-*` properties, no?

Reply:
It's handled by https://github.com/Siteimprove/alfa/blob/main/packages/alfa-style/src/property/border-top-width.ts#L48-L54 (and the previous test).

Reply:
Good point!